src/main/webapp/splash.html

```diff
@@ -17,15 +17,18 @@ <h2>Beetle</h2>
 
       <div name="text_enter" class="text-enter">
         <div id="select-db">
-          <form action="/splash.html" method="GET" id="database-select-form" >
+          <form action="/splash.html" method="GET; login()" id="database-select-form" >
```

Comment:
I haven't seen this before. How does it work? Should we use onsubmit for this? Don't we want to execute login() onload and not necessarily when the form is submitted?

Reply:
The login onload still occurs. I believed that this does work similarly to an onsubmit. When the databases are selected I want login to be called again before the tables are shown. It might be better to use an onsubmit in case this does not work how I expected